custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/chips/chip_mdi_icon_state.yaml

```diff
@@ -8,7 +8,17 @@ chip_mdi_icon_state:
   triggers_update: "all"
   show_icon: true
   icon: "[[[ return variables.ulm_chip_mdi_icon_state_icon ]]]"
-  label: "[[[ return states[variables.ulm_chip_mdi_icon_state_entity].state ]]]"
+  label: |
+    [[[
+      var state = "";
+      if (states[variables.ulm_chip_mdi_icon_state_entity].state){
+        var state = states[variables.ulm_chip_mdi_icon_state_entity].state;
+        if (states[variables.ulm_chip_mdi_icon_state_entity].attributes.unit_of_measurement){
+          state += states[variables.ulm_chip_mdi_icon_state_entity].attributes.unit_of_measurement;
+        }
+      }
+      return state;
+    ]]]
```

Comment:
Is this change intended to be part of this Pull Request?

Reply:
No, it's not. This is already part of #1037.
I'll do some reading to avoid these kind of conflicts in future. Sorry for that. Github PR is new to me, I need do dig in deeper to understand how to commit just the relevant lines.

Reply:
@shogun160 Okay I see. 
For both PR's you have used the same branch `shogun160:main` as develop branch. All changes in your local branch are added to all PR's you do with that branch. Especially when they are directed to different branches on the main project.
So it's advised to use either a new local branch for each PR or re-sync your fork with the main project and then start adding the changes 😉 
For now I will solve this with some commits to this PR also to fix upcoming conflicts with the other similar mentioned PR.

Reply:
@basbruss Thanks a lot!
Ok, I got it. Also Thank you for your explanation. I know how to deal with this in future, but will do some more reading to get a "bigger picture".